openassessment/xblock/openassessmentblock.py

```diff
@@ -1242,3 +1243,45 @@ def get_xblock_id(self):
         Returns the xblock id
         """
         return text_type(self.scope_ids.usage_id)
+
+    def _strip_html_tags_and_extra_whitespaces(self, raw_html):
+        cleanr = re.compile('<.*?>')
+        cleantext = str(re.sub(cleanr, '', raw_html))
+        cleantext = " ".join(re.split(r"\s+", cleantext, flags=re.UNICODE)).strip()
+        return cleantext
```

Comment:
[Same comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/244#discussion_r446313260) about using bleach

Reply:
Done